TAScheduler/views.py

```diff
@@ -277,6 +279,65 @@ def post(self, request):
                               {"message": str(e), "courses": courses})
 
 
+class InstructorAssignTaToSection(View):
+
+    def get(self, request):
+        if request.session.get("user") is None:
+            return redirect("/")
+        if determineUser(request.session["user"]).getRole() != "Instructor":
+            return redirect("/home/")
+        tas = list(map(str, TA.objects.all()))
+        sections = list(map(str, Lab.objects.all()))
+        sections.extend(list(map(str, Lecture.objects.all())))
+        if len(tas) == 0:
+            return render(request,
+                          "error.html",
```

Comment:
i don't think we have error.html anymore?  (I could be wrong, my version of the project is always messed up)